Assignment-4/app/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import "./App.css";
+import InputField from "./Components/InputField";
+import Form from "./Components/Form";
+import SpecificInputValue from "./Components/SpecificInputValue";
+import CheckPassword from "./Components/CheckPassword";
+import DropDownMenu from "./Components/DropDownMenu";
+import ToDoList from "./Components/ToDoList";
+import TempConverter from "./Components/TempConverter";
+import FormValidation from "./Components/FormValidation";
+import FormSubmitOrNot from "./Components/FormSubmitOrNot";
+import FormValidationRule from "./Components/FormValidationRule";
+import DataTable from './Components/DataTable'
+import SidebarNavigation from "./Components/SidebarNavigation";
+import FilterComponent from "./Components/FilterComponent";
+import ShowModel from './Components/ShowModel'
```

Comment:
Create a index file and import all the components from there

Reply:
ok